x/compliance/keeper/msg_server_provision_model.go

```diff
@@ -41,6 +41,10 @@ func (k msgServer) ProvisionModel(goCtx context.Context, msg *types.MsgProvision
 		}
 	}
 
+	if !types.IsValidPFCCertificationRoute(msg.ParentChild) {
```

Comment:
Please move this check to `validate` in message_certify_model (the same way as for `IsValidCertificationType`)

Reply:
Done! ✅